tests/dragonfly/cluster_test.py

```diff
@@ -570,14 +584,20 @@ async def test_cluster_blocking_command(df_server):
     await close_clients(c_master, c_master_admin)
 
 
+@pytest.mark.parametrize("set_cluster_node_id", [True, False])
 @dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
 async def test_cluster_native_client(
```

Comment:
where do we run replicaof in this test? I cant find it..

Reply:
good catch! :)
I added it, even though it doesn't really matter too much, because we never reach out to the replicas.. should we?

Reply:
We have this in the test.. I guess it is not connecting to the replica as we thought, as the test would have failed before because we did not really replicated
    # Make sure that getting a value from a replica works as well.
    replica_response = await client.execute_command(
        "get", "key0", target_nodes=aioredis.RedisCluster.REPLICAS
    )
    assert "value" in replica_response.values()

Reply:
again, very nice catch!
The reason why it worked before is that it followed `MOVED` error replies to the relevant master, and got the response from it.
Now I use the connections directly, which does not follow `MOVED` (I checked and it fails if I comment out the `replicaof` flag), and now the test works.